common/src/main/java/fi/hel/verkkokauppa/common/util/DateTimeUtil.java

```diff
@@ -53,6 +54,15 @@ public static LocalDateTime fromFormattedDateTimeString(String localDateTime) {
         return LocalDateTime.parse(localDateTime, formatter);
     }
 
+    public static LocalDateTime fromFormattedDateTimeOptionalString(String localDateTime) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
Could that pattern to be moved to constant and given name of the format used. Constant can be created to this class.

DateTimeUtil.DATE_TIME_WITH_MILLISECONDS etc

Reply:
Sure, I'll change this.